springfox-swagger-common/src/main/java/springfox/documentation/swagger/web/UiConfiguration.java

```diff
@@ -23,155 +23,157 @@
+   *                                 requests.
+   * @param docExpansion             Controls the default expansion setting for the operations and tags. It can be
+   *                                 'list' (expands only the tags), 'full' (expands the tags and operations) or 'none'
+   *                                 (expands nothing).
+   * @param filter                   If set, enables filtering. The top bar will show an edit box that you can use to
+   *                                 filter the tagged operations that are shown. Can be Boolean to enable or disable,
+   *                                 or a string, in which case filtering will be enabled using that string as the
+   *                                 filter expression. Filtering is case sensitive matching the filter expression
+   *                                 anywhere inside the tag.
+   * @param maxDisplayedTags         If set, limits the number of tagged operations displayed to at most this many. The
+   *                                 default is to show all operations.
+   * @param operationsSorter         Apply a sort to the operation list of each API. It can be 'alpha' (sort by paths
+   *                                 alphanumerically), 'method' (sort by HTTP method) or a function (see
+   *                                 Array.prototype.sort() to know how sort function works). Default is the order
+   *                                 returned by the server unchanged.
+   * @param showExtensions           Controls the display of vendor extension (x-) fields and values for Operations,
+   *                                 Parameters, and Schema.
+   * @param tagsSorter               Apply a sort to the tag list of each API. It can be 'alpha' (sort by paths
+   *                                 alphanumerically) or a function (see Array.prototype.sort() to learn how to write a
+   *                                 sort function). Two tag name strings are passed to the sorter for each pass.
+   *                                 Default is the order determined by Swagger-UI.
+   * @param validatorUrl             By default, Swagger-UI attempts to validate specs against swagger.io's online
+   *                                 validator. You can use this parameter to set a different validator URL, for example
+   *                                 for locally deployed validators (Validator Badge). Setting it to null will disable
+   *                                 validation. This parameter is relevant for Swagger 2.0 specs only.
    */
   public UiConfiguration(
-      String validatorUrl,
-      String docExpansion,
-      String apisSorter,
-      String defaultModelRendering,
-      String[] supportedSubmitMethods,
-      boolean jsonEditor,
-      boolean showRequestHeaders,
-      Long requestTimeout) {
-    this.validatorUrl = validatorUrl;
-    this.docExpansion = docExpansion;
-    this.apisSorter = apisSorter;
```

Comment:
@dilipkrish Should I fix this?

Reply:
No don't worry about it. It was my bad when I initially created and kept adding to it. 😢 

The builder should've been the initial choice from the beginning, not removing it will break compatibility. If you don't mind you could mark it as deprecated and point users to the Builder. and annotate that it will be removed in 3.0 of springfox.

Reply:
@dilipkrish Thanks. I will mark it as deprecated.